internal/beatcmd/beat.go

```diff
@@ -457,6 +484,176 @@ func (b *Beat) registerMetrics() {
 	monitoring.NewBool(managementRegistry, "enabled").Set(b.Manager.Enabled())
 }
 
+func (b *Beat) registerStatsMetrics() {
+	// TODO: we should ensure all metrics are produced in the expected JSON
```

Comment:
Is this done?

Reply:
The TODO was to make sure the metrics have the expected field names. I've fixed it in the other PR